src/v/kafka/requests/fetch_request.h

```diff
@@ -66,8 +66,8 @@ struct fetch_request final {
     model::node_id replica_id;
     std::chrono::milliseconds max_wait_time;
     int32_t min_bytes;
-    int32_t max_bytes;                                 // >= v3
-    int8_t isolation_level;                            // >= v4
+    int32_t max_bytes{-1};                             // >= v3
```

Comment:
in kafka this seems to be 0x7fffffff which i guess is like 2gb. Not sure it matters?